src/configs/chainconfig.json

```diff
@@ -0,0 +1,855 @@
+{
+    "1": "ETH",
```

Comment:
I was just expecting to use the top 5 most propular chains. Can you make it to just the top 5?

Reply:
That's fine.